### DISCUSSION TITLE
Custom notification sounds

### DISCUSSION THREAD

**Discussion by 97-42** [SethBurkart123/EvenBetterSEQTA#88] [Ideas]
To allow users to upload a mp3 that will function as the SEQTA notification sound.

**Comment by MEGA-Dawg68** [comment]
Is there ment to be code attached or are you asking for a feature?

**Reply by SethBurkart123** [reply]
I think just asking for a feature.

**Comment by SethBurkart123** [comment]
That seems like a cool idea.
